Sources/xcproj/PBXProjEncoder.swift

```diff
@@ -40,6 +40,7 @@ final class PBXProjEncoder {
         write(section: "PBXLegacyTarget", proj: proj, object: proj.objects.legacyTargets)
         write(section: "PBXProject", proj: proj, object: proj.objects.projects)
         write(section: "PBXResourcesBuildPhase", proj: proj, object: proj.objects.resourcesBuildPhases)
+        write(section: "PBXRezBuildPhase", proj: proj, object: proj.objects.carbonResourcesBuildPhases)
```

Comment:
Verified Xcode writes the PBXRezBuildPhase section between PBXResourcesBuildPhase and PBXShellScriptBuildPhase.

Reply:
:+1: